credsweeper/app.py

```diff
@@ -217,6 +225,14 @@ def file_scan(self, content_provider: ContentProvider) -> List[Candidate]:
             # Skip the file scanning and create fake candidate because the extension is suspicious
             candidates.append(Candidate.get_dummy_candidate(self.config, content_provider.file_path))
 
+        elif self.config.max_depth > 0 and isinstance(content_provider, TextContentProvider):
+            # Feature to scan files which might be containers
+            data = Util.read_data(content_provider.file_path)
+            if data:
+                data_provider = DataContentProvider(data=data, file_path=content_provider.file_path)
+                # use limit with 1Gb
+                candidates = self.data_scan(data_provider, self.config.max_depth, 1 << 30)
```

Comment:
Set limitation to 1Gb with `1 << 30` is good. But what about use `Constant` class?
I think it would be better adding `ZIP_SCAN_LIMITATION` attribute and using it.

Reply:
`RECURSIVE_SCAN_LIMITATION` has been added and resolve conversation.